firmware/library/L3_HAL/accelerometer_driver/accelerometer.hpp

```diff
@@ -0,0 +1,20 @@
+// accelerometer.hpp
+// Created on: Jun 20, 2018
+// Author: Andrew Javier

```

Comment:
You can actually remove all of these comment lines. This information will be known from the commit history, since you are the sole author of this file.

The name of the file is known from the file name itself.